netplugin/netd.go

```diff
@@ -76,6 +77,11 @@ func startNetPlugin(pluginConfig *plugin.Config) {
 }
 
 func initNetPluginConifg(ctx *cli.Context) (plugin.Config, error) {
```

Comment:
if you end up with another push, please fix the spelling here as a drive by?

Reply:
Sure, I'll fix it in a separate commit.